src/app/myworkflows/my-workflow/my-workflow.component.html

```diff
@@ -88,13 +87,15 @@
               [refreshMessage]="isRefreshing$ | async"
             ></app-sidebar-accordion>
             <br />
-            <div class="clearfix sidebar-header">
-              <h3 class="sidebar-header">Shared with me</h3>
-            </div>
-            <p *ngIf="groupSharedEntriesObject == undefined || groupSharedEntriesObject == null || groupSharedEntriesObject.length == 0">
-              You have no {{ entryType$ | async }}s shared with you
-            </p>
-            <app-sidebar-accordion [openOneAtATime]="oneAtATime" [groupEntriesObject]="groupSharedEntriesObject"></app-sidebar-accordion>
+            <ng-container *ngIf="entryType !== EntryType.Service">
+              <div class="clearfix sidebar-header">
+                <h3 class="sidebar-header">Shared with me</h3>
+              </div>
+              <p *ngIf="groupSharedEntriesObject == undefined || groupSharedEntriesObject == null || groupSharedEntriesObject.length == 0">
```

Comment:
I would personally have `!groupSharedEntriesObject` vs. the two explicit comparisons, but you can argue it either way.

Reply:
Hide whitespace changes.  Though I do agree.

Reply:
I think I lost that setting when I moved from Chrome to FireFox, or at least that's my excuse.